packages/twenty-front/src/modules/object-record/record-table/record-table-body/components/RecordTableBodyRecordGroupDragDropContext.tsx

```diff
@@ -0,0 +1,108 @@
+import { DragDropContext, DropResult } from '@hello-pangea/dnd';
+import { ReactNode, useContext } from 'react';
+import { useRecoilCallback, useSetRecoilState } from 'recoil';
+
+import { useUpdateOneRecord } from '@/object-record/hooks/useUpdateOneRecord';
+import { recordGroupDefinitionFamilyState } from '@/object-record/record-group/states/recordGroupDefinitionFamilyState';
+import { recordIndexAllRecordIdsComponentSelector } from '@/object-record/record-index/states/selectors/recordIndexAllRecordIdsComponentSelector';
+import { RecordTableContext } from '@/object-record/record-table/contexts/RecordTableContext';
+import { useComputeNewRowPosition } from '@/object-record/record-table/hooks/useComputeNewRowPosition';
+import { isRemoveSortingModalOpenState } from '@/object-record/record-table/states/isRemoveSortingModalOpenState';
+import { useRecoilComponentCallbackStateV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentCallbackStateV2';
+import { getSnapshotValue } from '@/ui/utilities/state/utils/getSnapshotValue';
+import { useGetCurrentView } from '@/views/hooks/useGetCurrentView';
+import { isDefined } from '~/utils/isDefined';
+
+export const RecordTableBodyRecordGroupDragDropContext = ({
```

Comment:
I feel the suffix "Context" is not right, I would expect it to be a basic react Context. Maybe ContextProvider (which is not perfect too as Context.Provider is also something :p )